src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrRuntimeInstance.cs

```diff
@@ -89,6 +91,27 @@ internal DkmClrType GetType(string typeName, params System.Type[] typeArguments)
             return null;
         }
 
+        private IEnumerable<DkmClrModuleInstance> WithMscorlibLast(DkmClrModuleInstance[] list)
+        {
+            DkmClrModuleInstance mscorlib = null;
+            foreach (var module in list)
+            {
+                if (module.Assembly.GetReferencedAssemblies().Length == 0)
```

Comment:
I had tried that, but they are actually different `Assembly` types.
One is `System.Reflection.Assembly` and the other is `Microsoft.VisualStudio.Debugger.Metadata.Assembly`.